bedrock/firefox/templates/firefox/pocket.html

```diff
@@ -0,0 +1,116 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block page_title %}{{ _('Pocket') }}{% endblock %}
+{% block body_id %}firefox-pocket{% endblock %}
+{% block body_class %}
+  {{ super() }}
+  mozid
```

Comment:
Hmm, this won't conflict - but we should tread carefully (cc @jpetto), as there's a `mozID` body class being added to sandstone in another PR here: https://github.com/mozilla/bedrock/pull/2932/files#diff-8a894a393259a3ba520a09de584b6d1dR174.

It looks like the [family page](https://github.com/mozilla/bedrock/blob/master/media/css/firefox/family/index.less#L7) also has a body class (lowercase) of mozid. I don't think anything should break here or conflict due to case sensitivity, but we should probably be standardize this in some way.